pandas/core/arrays/datetimelike.py

```diff
@@ -80,6 +81,26 @@
 DatetimeLikeArrayT = TypeVar("DatetimeLikeArrayT", bound="DatetimeLikeArrayMixin")
 
 
+def ravel_compat(meth):
```

Comment:
this actually could be in a more general location, maybe pandas/core/array_alogs/ somewhere?

Reply:
maybe core.arraylike?  For now im just excited about getting `__array_function__` working on NDArrayBackedExtensionArray.

Reply:
sure

Reply:
oh, wait, the implementation here is specific to NDArrayBackedExtensionArray, will move it there

Reply:
moved + green